### DISCUSSION TITLE
Edit

### DISCUSSION THREAD

**Discussion by ylcndgn** [IBAX-io/go-ibax#13] [Typos （English）]
Member information
Wallet Address: 1116-1730-8569-4061-1808
Balance: 10000 IBXC
Transactions historySend Conis---->(coins)
Deposit: 0 IBXC
Roles: 
none
Request

**Comment by HalloKyle** [comment]
> Member information Wallet Address: 1116-1730-8569-4061-1808 Balance: 10000 IBXC Transactions historySend Conis---->(coins) Deposit: 0 IBXC Roles: none Request


Thank you for your kindly support will ask our tech guy to take a look at it as soon as possible. Kindly ask if you could star and Fork our project that will be very helpful for us to grow. and you will be rewarded 500 IBAX points. Thank you!
